packages/netflow/manifest.yml

```
@@ -1,9 +1,9 @@
 format_version: 1.0.0
 name: netflow
-title: NetFlow
-version: 1.2.2
+title: NetFlow Records
+version: 1.2.3
 license: basic
-description: Collect logs from NetFlow with Elastic Agent.
+description: Collect records from NetFlow and IPFIX flow with Elastic Agent.
```

Suggestion:

```suggestion
description: Collect flow records from NetFlow and IPFIX exporters with Elastic Agent.
```

The software the sends NetFlow/IPFIX data is generally referred to an "exporter".

---

Thanks for the review @andrewkroh. That looks good to me.